README.md

```diff
@@ -94,6 +94,12 @@ mvn clean package -DskipTests -Dspark3.1.x
 
 Starting from versions 0.11, Hudi no longer requires `spark-avro` to be specified using `--packages`
 
+### Usage of hudi-utilities-slim-bundle
+
+Starting from versions 0.11, Hudi provides hudi-utilities-slim-bundle which excludes hudi-spark-datasource modules.
+This new bundle is intended to be used with Hudi Spark bundle together, if using hudi-utilities-bundle solely
+introduces problems for a specific Spark version.
+
```

Comment:
as discussed we can move this to packages' own readme instead in the repo readme

Reply:
Addressed in #5184